### ISSUE TITLE
Entire site A11Y: Redundant links, plus lack of descriptive link text in "Learn More" links

### ISSUE THREAD

**Issue by josieg** [BCcampus/open-sage#143]
**Describe the bug**
Throughout the site, many pages have multiple "Learn More" button links. These links need descriptive link text (or unique aria labels). I also have doubts about linking to both the headings AND the "learn more" buttons, because that seems redundant.

**To Reproduce**
Learn More links on the following pages: 
- https://helga.bccampus.ca/wp/opentextbooks/ 
- https://helga.bccampus.ca/wp/opentextbooks/what-is-open-education/
- https://helga.bccampus.ca/wp/opentextbooks/2-0-find-open-textbooks/
- https://helga.bccampus.ca/wp/opentextbooks/use-open-textbooks/
- https://helga.bccampus.ca/wp/opentextbooks/create-open-textbooks/

**Expected behavior**
I would expect that a page would only have one link to the same place and that links are descriptive.

I would propose removing links from the headings/images and adding descriptive aria-labels to all of the "Learn More" buttons so they are unique. For example, they could match the heading (i.e. FIND OPEN TEXTBOOKS) or be longer (i.e. LEARN HOW TO CREATE AN OPEN TEXTBOOK).

![homepage](https://user-images.githubusercontent.com/23706762/54724202-644bae80-4b27-11e9-8407-63755170e747.jpg)

**Comment by alex-418** [comment]
@josieg We discussed this with @ehbeattie and decided to keep the link text as "learn more" but update what the screen reader text would say to include what the heading says.

**Comment by alex-418** [comment]
Updated these pages via https://github.com/BCcampus/open-sage/commit/de30e3388055240c88375f082592f4303653cbc8, plus on advocate for open education, and projects/grants page to include the following aria-label on the "learn more" buttons: 

`aria-label="Read more about (title of the page is here)"`

**Comment by ehbeattie** [comment]
Perfect.
